docs/src/main/sphinx/release/release-458.md

```diff
@@ -0,0 +1,108 @@
+# Release 458 (16 Sep 2024)
+
+## General
+
+* Improve performance for queries with a redundant `DISTINCT` clause. ({issue}`23087`)
+
+## ClickHouse connector
+
+* Reduce data transfer from remote systems for queries with large `IN` lists. ({issue}`23381`)
```

Comment:
Please skip this, the default for ClickHouse was already customized to a higher value before